tests/imodels-clients-tests/src/integration/management/IModelOperations.test.ts

```diff
@@ -43,6 +44,7 @@ describe("[Management] IModelOperations", () => {
 
     const testIModelCreator = container.get(TestIModelCreator);
     testIModelForUpdate = await testIModelCreator.createEmpty(testIModelGroup.getPrefixedUniqueIModelName("Test iModel for update"));
+    testIModelForSearch = await testIModelCreator.createEmpty(testIModelGroup.getPrefixedUniqueIModelName("Test iModel for search"));
```

Comment:
could we reuse `testIModelForUpdate`? the more iModels we create for tests the more flaky they'll be, so let's avoid that whenever possible.

Reply:
updated